hdl/projects/grapefruit/grapefruit_timing.xdc

```diff
@@ -67,4 +67,18 @@ set_output_delay -clock fmc_virt_clk -min 0.830 [get_ports fmc_sp_to_fpga_da[*]]
 set_multicycle_path -from [get_pins {stm32h7_fmc_target_inst/data_out*/C}] -to [get_ports {fmc_sp_to_fpga_da[*]}] -setup 2
 set_multicycle_path -from [get_pins {stm32h7_fmc_target_inst/data_out*/C}] -to [get_ports {fmc_sp_to_fpga_da[*]}] -hold 1
 set_multicycle_path -from [get_pins {stm32h7_fmc_target_inst/data_out_en_reg*/C}] -to [get_ports {fmc_sp_to_fpga_da[*]}] -setup 2
-set_multicycle_path -from [get_pins {stm32h7_fmc_target_inst/data_out_en_reg*/C}] -to [get_ports {fmc_sp_to_fpga_da[*]}] -hold 1
\ No newline at end of file
+set_multicycle_path -from [get_pins {stm32h7_fmc_target_inst/data_out_en_reg*/C}] -to [get_ports {fmc_sp_to_fpga_da[*]}] -hold 1
+
+# #######################
+# FMC Interface
+# #######################
+
+# in delay max = tco_ext to max delay ext to fpga
+# in delay min = minTco_ext to min delay ext to fpga
+
+
+# Create a virtual clock, to represent the source clock of the eSPI interface
+#create_clock -name espi_virt_clk -period 50.000;  # 20 MHz clock
+
+#set_output_delay -clock espi_virt_clk -max 3.8 [get_ports espi_hpm_to_scm_dat[*]]
+#set_output_delay -clock espi_virt_clk -min 0.8 [get_ports espi_hpm_to_scm_dat[*]]
```

Comment:
Looks like cleaning this up was maybe forgotten?

Reply:
Yeah, I don't have this properly constrained yet, so removed for now so as not to confuse anything.